vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -29,7 +29,7 @@ Configure a Keycloak client to grant the bundler access to the required Entando
 
 1. Log in to Keycloak using the admin credentials. The URL will be similar to `http://YOUR-HOST-NAME/auth/` and can be verified with the following command:
 ``` sh
-kubectl describe ingress/default-sso-in-namespace
+ent kubectl describe ingress/default-sso-in-namespace-ingress
```

Comment:
Let's leave this as kubectl and add the missing -n entando. Looking at the rest of the tutorial, I think that's clearest.